conda/repodata.py

```diff
@@ -0,0 +1,99 @@
+# (c) Continuum Analytics, Inc. / http://continuum.io
```

Comment:
I think there are several very good options we have to indicate what's API and what's not.  I might suggest a starting point of, whatever is available with

    from conda.api import *

and then take it from there.